conf/metrics.go

```diff
@@ -0,0 +1,26 @@
+package conf
+
+type MetricsExporter = string
+
+const (
+	Prometheus           MetricsExporter = "prometheus"
+	OpenTelemetryMetrics MetricsExporter = "opentelemetry"
+)
+
+type MetricsConfig struct {
+	Enabled bool
+
+	Exporter MetricsExporter `default:"opentelemetry"`
+
+	// ExporterProtocol is the OTEL_EXPORTER_OTLP_PROTOCOL env variable,
+	// only available when exporter is opentelemetry. See:
+	// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md
+	ExporterProtocol string `default:"http/protobuf" envconfig:"OTEL_EXPORTER_OTLP_PROTOCOL"`
+
+	PrometheusListenHost string `default:"0.0.0.0"`
+	PrometheusListenPort string `default:"9100"`
```

Comment:
Thinking of using the default OTEL variables for this as specified here: https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#prometheus-exporter